packages/pancake-uikit/src/components/PancakeToggle/StyledPancakeToggle.tsx

```diff
@@ -0,0 +1,175 @@
+import styled from "styled-components";
+import { scales, PancakeToggleProps, HandleProps, InputProps, ScaleKeys } from "./types";
+
+const scaleKeyValues = {
```

Comment:
Can you put a comment in here explaining these values a bit.

Reply:
@hachiojidev pushed new commit with comments. Also renamed `handleSize` to `pancakeSize` to be more consistent with naming here.

Reply:
@hachiojidev also had to rebase on master (test was failing), force pushed rebased branch